kimchi/src/lagrange_basis_evaluations.rs

```diff
@@ -240,7 +245,6 @@ mod tests {
                     Fp::zero()
                 });
             }
-            e = vec![Fp::zero(), Fp::one()];
```

Comment:
oh, I wonder why this was here... Basically this was reverting the lines above 😞